subprojects/android/android.gradle

```diff
@@ -3,8 +3,31 @@ description = "Mockito for Android"
 apply from: "$rootDir/gradle/java-library.gradle"
 
 dependencies {
-    compile project.rootProject
+    compile(project.rootProject) {
+        exclude group: 'org.objenesis', module: 'objenesis'
+    }
     compile libraries.bytebuddyandroid
+    compile(libraries.objenesis2) {
+        version {
+            strictly '2.6'
+        }
+        because(
+            '\n' +
+                '--------------------------------------\\     /-------------------------------------\n' +
+                '---------------------------------------\\   /--------------------------------------\n' +
+                '----------------------------------------\\ /---------------------------------------\n' +
+                '-----------------------------------------V----------------------------------------\n' +
+                'Objenesis 3.x version has method that is not supported by android api 25 and below.\n' +
+                'If you have mockito-android or mockito-kotlin, exclude mockito-core from them.\n' +
+                'implementation("com.nhaarman.mockitokotlin2:mockito-kotlin") {\n' +
+                '    exlcude group: "org.mockito", module: "mockito-core"\n' +
+                '}\n' +
```

Comment:
```suggestion
                'Objenesis 3.x does not work with android api 25 and below.\n' +
                'If you have mockito-android or mockito-kotlin, exclude mockito-core from them.\n' +
                'implementation("com.nhaarman.mockitokotlin2:mockito-kotlin") {\n' +
                '    exlcude group: "org.mockito", module: "mockito-core"\n' +
                '}\n' +
```

Reply:
Updated text and added pr and issue links for further details.